### DISCUSSION TITLE
Getting Started: I have 3 branches each branched from the previous. How do i move them into a stack?

### DISCUSSION THREAD

**Discussion by markalanevans** [gitext-rs/git-stack#354] [General]
Given i have 3 branches, a, b, c each is branched from the previous, can i create a stack out of them? FYI these were created before i had installed git-stack.

**Comment by epage** [comment]
Stacks are implicitly managed in `git stack`, so having existing branches is not a problem. This lower overhead / better interop comes at the cost that we can't always tell what was intended.
